src/components/contact-form.js

```diff
@@ -0,0 +1,33 @@
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+} from "@chakra-ui/react";
+
+export default function ContactForm() {
+  const handleOnSubmit = (event) => {
+    event.preventDefault();
+  };
+
+  return (
+    <form onSubmit={handleOnSubmit}>
+      <FormControl isRequired>
+        <FormLabel htmlFor="email">Email address</FormLabel>
+        <Input id="email" type="email" placeholder="example@mail.com" />
+      </FormControl>
+      <br />
```

Comment:
Don't use `<br />` to break content, rather use CSS for this purpose. Flex is a great way to do this with `flex-gap`. You can see elsewhere how others did this with `spacing` (which `@chakra-ui` supports). Here's the docs for this as well:
https://chakra-ui.com/docs/layout/stack#import